run_test_specific.py

```
@@ -0,0 +1,11 @@
+import os
+os.environ['SECRET_KEY'] = 'dummy_key'
```

Ruff (0.15.7)

[error] Possible hardcoded password assigned to: "SECRET_KEY"

(S105)